src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Configurations/ElasticBeanstalkManagedPlatformUpdatesConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AspNetAppElasticBeanstalkLinux.Configurations
+{
+    public class ElasticBeanstalkManagedPlatformUpdatesConfiguration
+    {
+        public bool ManagedActionsEnabled { get; set; } = true;
+        public string PreferredStartTime { get; set; } = "Sun:00:00";
```

Comment:
Can we use DateTime?

Reply:
I don't see a reason to do so. We don't support this object type in the cli and we need to pass it to CDK as string as well.